.github/workflows/ci-ruby.yml

```diff
@@ -47,7 +47,7 @@ jobs:
             os: macos
           - ruby-version: 3.3.5
             os: ubuntu
-          - ruby-version: jruby-9.4.8.0
+          - ruby-version: jruby-9.4.9.0
```

Comment:
I think it's also used in `ci-rbe.yml`.

Reply:
I just searched and replace for all the 9.4.8.0 and I make a note for next version upgrade and all the files we use it